tutorials/metrics/plot_metrics.py

```diff
@@ -30,88 +48,143 @@
 from pycrostates.cluster import ModKMeans
 from pycrostates.datasets import lemon
 
+
 raw_fname = lemon.data_path(subject_id="010017", condition="EO")
 raw = read_raw_eeglab(raw_fname, preload=True)
 raw.crop(0, 30)
-
 raw.pick("eeg")
 raw.set_eeg_reference("average")
 
-#%%
-# Pycrostates implements several metrics to evaluate the quality of
-# clustering solutions without knowing the ground truth:
-# - The silhouette score (higher the better)
-# - The Calinski Harabasz score (higher the better)
-# - The Dunn score (higher the better)
-# - The Davies Bouldin score (lower the better)
-#
-# We can apply these metrics on clustering solution
-# computed with different values of n_clusters and
-# analyse which give the best clustering solution.
+# %%
+# In this example, we will use a single subject EEG recording in order to give
+# more explanation about each of this metrics before showing an application
+# case to determine the optimal number of :term:`cluster centers`
+# ``n_clusters`` while fitting a :class:`~pycrostates.cluster.ModKMeans`
+# algorithm.
+# For this example, we start by computing each of the score on the clustering
+# results of a :class:`~pycrostates.cluster.ModKMeans` fitted for different
+# ``n_clusters`` ranging from 2 to 8.
+
 from pycrostates.metrics import (
     silhouette_score,
     calinski_harabasz_score,
     dunn_score,
     davies_bouldin_score,
 )
 
-cluster_numbers = range(2,10)
-silhouette_scores = []
-calinski_harabasz_scores = []
-dunn_scores = []
-davies_bouldin_scores = []
-
-for k in cluster_numbers:
-    ModK = ModKMeans(n_clusters=k, random_state=42)
+cluster_numbers = range(2, 9)
+scores = dict(silhouette=[], calinski_harabasaz=[], dunn=[], davies_bouldin=[])
+for n_clusters in cluster_numbers:
+    # fit K-means algorithm with a set number of cluster centers
+    ModK = ModKMeans(n_clusters=n_clusters, random_state=42)
     ModK.fit(raw, n_jobs=2)
-    # silhouettee
-    silhouette_scores.append(silhouette_score(ModK))
-    # calinski and harabasz
-    calinski_harabasz_scores.append(calinski_harabasz_score(ModK))
-    # dunn
-    dunn_scores.append(dunn_score(ModK))
-    # davies bouldin
-    davies_bouldin_scores.append(davies_bouldin_score(ModK))
+
+    # compute scores
+    scores["silhouette"].append(silhouette_score(ModK))
+    scores["calinski_harabasaz"].append(calinski_harabasz_score(ModK))
+    scores["dunn"].append(dunn_score(ModK))
+    scores["davies_bouldin"].append(davies_bouldin_score(ModK))
 
 #%%
 # Silhouette score
+# ----------------
+# The Silhouette score\ :footcite:p:`Silhouettes` focuses on 2 metrics: the
+# intra-cluster distance and the inter-cluster distance. It summarizes how well
+# clusters are dense and well separated.
```

Comment:
All scores are based on intra-cluster distance and inter-cluster distance. I think it would be nice to define those terms before we use them, ideally with an illustration. @vferat By any chance, do you have an illustration for that?

Reply:
By anychance, did you not delete the images from your computer? 
Can you also upload the one for the darkmode?

![image](https://user-images.githubusercontent.com/73893616/192726875-4225463b-55dd-43cb-b328-f193457e12f5.png)

The background is pure black: 

![image](https://user-images.githubusercontent.com/73893616/192727265-fb1cb646-24cb-4541-bda4-8cb2770addab.png)

Reply:
Thanks :)